scripts/generate_pipeline_files.py

```diff
@@ -0,0 +1,162 @@
+)
+def create_pipeline(dataset_id: str, pipeline_id: str):
+    new_pipeline_path = f"{DATASETS_PATH}/{dataset_id}/pipelines/{pipeline_id}/"
+    dir = os.path.dirname(new_pipeline_path)
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+        click.echo(
+            f"\n{DATASETS_PATH}/{dataset_id}/pipelines/{pipeline_id} has been created\n"
+        )
+    else:
+        click.echo(f"\n{new_pipeline_path} already exists\n")
+    create_dataset_yaml(dataset_id)
+    create_pipeline_yaml(dir)
+
+
+def create_dataset_yaml(dataset_id: str):
+    output = {}
+    sample_yaml = yaml.load((PROJECT_ROOT / "samples" / "dataset.yaml").read_text())
+    sample_yaml["dataset"]["name"] = dataset_id
+    sample_yaml["dataset"]["friendly_name"] = dataset_id
+    dataset_desc = click.prompt("A user-friendly description of the dataset", type=str)
+    sample_yaml["dataset"]["description"] = dataset_desc
+    output["dataset"] = sample_yaml["dataset"]
+
+    resources = []
+    if click.confirm("\nWill you need GCP Resource(s) for your pipeline?"):
+        while True:
+            resource = click.prompt(
+                (
+                    "\nWhich GCP Resource(s) are required for your pipeline\n"
+                    "Select Resources Needed: BigQuery (BQ), Google Cloud Storage (GCS)?"
+                ),
+                type=click.Choice(["BQ", "GCS"], case_sensitive=False),
+                default="r",
+            )
+            if resource == "BQ":
+                bq_yaml = sample_yaml["resources"][0]
+                bq_yaml["dataset_id"] = dataset_id
+                bq_desc = click.prompt(
+                    "\nA user-friendly description of the dataset", type=str
+                )
+                bq_yaml["description"] = bq_desc
+                resources.append(bq_yaml)
+            if resource == "GCS":
+                gcs_yaml = sample_yaml["resources"][1]
+                gcs_bucket_name = click.prompt(
+                    "\nYour Cloud Storage Bucket Name\n"
+                    "Use hyphenated syntax, e.g. `some-prefix-123`, for the names.\n"
+                    "Note that bucket names must not contain 'google' or close misspellings, such as 'g00gle'.",
+                    type=str,
+                )
+                location = click.prompt(
+                    (
+                        "\nThe location of the bucket.\n"
+                        "Object data for objects in the bucket resides in physical storage within this region.\n"
+                        "Defaults to US."
+                    ),
+                    type=click.Choice(["US", "EU", "ASIA"], case_sensitive=False),
+                    default="US",
+                )
+                gcs_yaml["name"] = gcs_bucket_name
+                gcs_yaml["location"] = location
+                resources.append(gcs_yaml)
+            if not click.confirm("\nWould you like to add another resource"):
+                output["resources"] = resources
```

Comment:
`output` can be an overloaded term. Use another name like `dataset_yaml` to mean that the dict will map to the dataset.yaml file